util/all_tests.json

```diff
@@ -27,6 +27,12 @@
     "env": ["OPENSSL_armcap=0x3D"],
     "target_arch": "arm"
   },
+  {
+    "comment": "Test OPENSSL_ia32cap on crypto_test for x86, as urandom_test is disabled for shared builds on x86",
+    "cmd": ["crypto/crypto_test"],
```

Comment:
This duplicates all the crypto_tests and takes a lot of time. Is there a single test we can execute that relies on OPENSSL_ia32cap that this can run with a `--gtest_filter`.